tuner/examples/simple/simple_tuner.py

```diff
@@ -19,6 +19,9 @@ def __init__(self, tuner_context: libtuner.TunerContext):
     def get_iree_compile_flags(self) -> list[str]:
         return self.compile_flags
 
+    def get_iree_compile_timeout_s(self) -> int:
```

Comment:
We should make this a field. That we update in the depending on whether we compile dispatches or models. For dispatches, 10s is reasonable, but for models a good starting point may be something like 60s.